README.md

```diff
@@ -110,6 +110,26 @@ toggleThemeMode();
 switchThemeScheme(ThemeMode.dark);
 ```
 
+
+# Use icons
+We use remix-icons. You need to run this script to scrap the svgs data from remix-icons. (This script is run automatically on `postinstall`, run this in case the icons aren't generated properly)
```

Comment:
```suggestion
We use remix-icons. You need to run this script to scrap the svgs data from remix-icons. (This script runs automatically on `postinstall`. Run this in case the icons aren't generated properly)
```